src/api/users.ts

```diff
@@ -12,3 +12,11 @@ type UserData = Pick<User, 'name' | 'email'>;
 export const createUser = async ({ email, name }: UserData) => {
   return client.post<User>('/users', { email, name });
 };
+
+export const getUserId = (user: User) => {
+  if (!user?.id) {
```

Comment:
user parametr is required so it can't be undefined

Reply:
Done ✅